lib/web_ui/test/mobile_unit_test_bindings.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:html' as html;
+import 'dart:js';
+import 'dart:js_util' as js_util;
+
+class MobileUnitTestBindings {
+  final Completer<bool> _allTestsPassed = Completer<bool>();
+
+  void tearDown() {
+    if (!_allTestsPassed.isCompleted) {
+      _allTestsPassed.complete(true);
```

Comment:
I only see the completer being completed to `true`. What happens when a test fails?